tests/signals.expect

```diff
@@ -12,6 +12,22 @@ send_line read
 expect -re "\\r\\n?read> $"
 exec -- kill -INT $pid
 expect "fish_postexec spotted"
+expect_prompt
+
+# Verify that the fish_kill_signal is set.
+send_line "functions -e postexec; function postexec --on-event fish_postexec; echo fish_kill_signal \$fish_kill_signal; end"
+expect_prompt
+send_line "sleep 5"
+sleep 0.100
+exec -- killall -INT sleep
```

Comment:
Let's do a `pkill -INT sleep -P $pid` to not kill unrelated processes.

Reply:
Done.